kornia/color/rgb.py

```diff
@@ -2,6 +2,52 @@
 import torch.nn as nn
 
 
+class RgbToRgba(nn.Module):
+    r"""Convert image from RGB to RGBA.
+
+    Add an alpha channel to existing image of special value named alpha_val
+
+    args:
+    image (torch.Tensor): A Three Channeled Image Tensor
+    alpha_val (int): An integer between 0 to 255
+    returns:
+            torch.Tensor: RGBA version of the image.
+
+    shape:
+        - image: :math:`(*, 3, H, W)`
+        - output: :math:`(*, 4, H, W)`
+
+    Examples::
+
+        >>> input = torch.rand(2, 3, 4, 5)
+        >>> rgba = kornia.color.RgbToRgba(alpha_val)
+        >>> output = rgba(input)  # 2x4x4x5
+
+   """
+    def __init__(self) -> None:
+
+        super(RgbToRgba, self).__init__()
+
+    def forward(self, image: torch.Tensor, alpha_val: float) -> torch.Tensor:
```

Comment:
couple of comments:
1. the alpha value would go to the constructor of the module, otherwise when you concatenate ops in a stack of sequence values you will face issues.
2. add `# type: ignore` at the end in order to avoid mypy errors